embedded-graphics/src/pixelcolor/mod.rs

```diff
@@ -108,5 +109,49 @@ pub trait PixelColor: Copy + PartialEq {
     /// See the [`raw` module documentation] for more details.
     ///
     /// [`raw` module documentation]: raw/index.html
-    type Raw: raw::RawData;
+    type Raw: RawData;
+}
+
+/// Convert a [`PixelColor`] into its underlying storage type
+///
+/// This trait provides the `into_storage()` method for implementors of [`PixelColor`]. This method
+/// exposes the underlying storage value of a pixel color type.
+///
+/// # Examples
+///
+/// ## Get the `u16` representing an `Rgb565` color
+///
+/// The [`Rgb565`] color type is stored in a `u16` internally. This example converts an [`Rgb565`]
```

Comment:
Isn't the first part of this paragraph redundant? The second sentence already states that the color is represented by a `u16`.

Reply:
Ah yep, just some copypasta noise. Removed :)